### PR TITLE
:lipstick: Header

### PR DESCRIPTION
**Pull request by mphelps1978**
Fixes issue  #11 - Request for Header
The logo is a temporary solution I found on google images, and modified to meet color theming requirements.

Moved the Hamburger menu to the right side, so that it was not interfering with the logo.
 
Removed the contributors link from footer component to inside the navBar, as per the issue requirement.

Adjusted styling using mui makeStyles attributes

### CODE REVIEW

**Comment by mphelps1978** [comment]
Preview of new NavBar: 
Screenshot_222

**Comment by MisterSingh** [comment]
Thanks, @mphelps1978 for contributing. Currently occupied with some prior work but will try to review the PR by EOD.

File: src/Components/Footer.js [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+          <a
+            style={{ textDecoration: "none" }}
+            href="https://github.com/Strategizer/app"
+          >
+            Open Source
+          </a>
+        </Typography>
+      </span>
+      {
+        //<span  style={{float : 'right'}}>
```
Comment (MisterSingh) [inline src/Components/Footer.js]: If just commenting this part is the only change in this file then please revert it (this file shouldn't be in the PR) as there is already an issue created for Footer that is already assigned to someone

Comment (mphelps1978) [reply src/Components/Footer.js]: Ooops.. I took that out to check the placement of the Contributors section of the header.. meant to put it back in.. fixing now

**Comment by MisterSingh** [comment]
@mphelps1978 Can you please also resolve the conflicts 😅. Just sync with master so that we can merge the PR

**Comment by mphelps1978** [comment]
Somehow, and I have no idea how.. I can't trace it back.. a new branch was created called 'header' as opposed to 'Header' - That one has the correct changes on it. I will close this PR, and open a new one with the correct branch..

**Pull request closed without merging**